scripts/deploy_nightly.sh

```diff
@@ -0,0 +1,38 @@
+
+git remote add mathlib "https://$GITHUB_TOKEN@github.com/leanprover-community/mathlib.git"
+git remote add nightly "https://$GITHUB_TOKEN@github.com/leanprover-community/mathlib-nightly.git"
+git fetch nightly --tags
+
+export MATHLIB_VERSION_STRING="nightly-$(date -u +%F)"
+
+if command -v greadlink >/dev/null 2>&1; then
+    # macOS readlink doesn't support -f option
+    READLINK=greadlink
+else
+    READLINK=readlink
+fi
+
+# Travis can't publish releases to other repos (or stop mucking with the markdown description), so push releases directly
+export GOPATH=$($READLINK -f go)
+PATH=$PATH:$GOPATH/bin
+go get github.com/itchio/gothub
+
+HEAD=`git rev-parse --abbrev-ref HEAD`
+if [ $HEAD = "master" ] && git tag $MATHLIB_VERSION_STRING
+then
+    last_tag=$(git describe @^ --abbrev=0 --tags)
```

Comment:
is this being used?

Reply:
It is for a feature linked to the description. I forgot to complete it. I took a lot of this from Lean and I left out the description. I'll put it in.